src/kirby/components/list/list.component.scss

```diff
@@ -35,11 +35,14 @@ ul {
         background-color: transparent;
       }
     }
+
+    &.clickable {
```

Comment:
Maybe the class should also be renamed. Sorry missed that in the last review.

Reply:
We can do that. But this class is purely for web. If you think I will :)

Reply:
*will rename it

Reply:
Do it 🚀 

But it is just to use the same  vocabulary in code and styling.